element_deeplabcut/model.py

```diff
@@ -646,8 +636,6 @@ def get_trajectory(cls, key, body_parts="all"):
                     and x/y coordinates of each joint name for a camera_id,

```

Comment:
Here `body_parts` is specified is a list but in the body of the function, it's compared against the string `'all'`. Update the docstring to allow for `'all'`.